nixos/cosmic/module.nix

```diff
@@ -61,6 +61,7 @@ in
       hicolor-icon-theme
       pop-icon-theme
       pop-launcher
+      playerctl
```

Comment:
```suggestion
      playerctl
      pop-icon-theme
      pop-launcher
```

Can you lexicographically sort these?

Reply:
Done! Sorry I had not noticed it was sorted, as everything started with `cosmic` x)